go/lang/security/filepath-clean-misuse.yaml

```diff
@@ -0,0 +1,40 @@
+rules:
+- id: filepath-clean-misuse
+  message: >-
+    `Clean` is not intended to sanitize against path traversal attacks.
+    This function is for finding the shortest path name equivalent to the given input.
+    Using `Clean` to sanitize file reads may expose this application to
+    path traversal attacks, where an attacker could access arbitrary files on the server.
+    To fix this easily, write this: `filepath.FromSlash(path.Clean("/"+strings.Trim(req.URL.Path, "/")))`
```

Comment:
opportunity for autofix?  Does autofix work with taint?

Reply:
It should, yeah, let me try it out